tools/Customization/DevHome.Customization/Views/MainPage.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.ObjectModel;
+using DevHome.Common;
+using DevHome.Common.Extensions;
+using DevHome.Common.Services;
+using DevHome.Customization.ViewModels;
+using DevHome.Settings.Models;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+
+namespace DevHome.Customization.Views;
+
+public sealed partial class MainPage : ToolPage
+{
+    public override string ShortName => "Windows customization";
```

Comment:
Does this string need to be localized?

Reply:
Good question @krschau / @EricJohnson327: none of these are localized across the project, what's the usage for ShortName?

Reply:
I was just asking @EricJohnson327 this on Friday. We think it's left over from an earlier version of navigation. It doesn't have to be localized because it's not used anymore -- instead, the text in the left nav comes from the resw file. At some point we should remove ShortName altogether.